src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/GroupedContextMenuStripTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace System.Windows.Forms.Design.Tests;
+
+public class GroupedContextMenuStripTests
+{
+    [Fact]
+    public void Populate_WhenInitializedWithoutGroups_ItemsCountIsZero()
+    {
+        GroupedContextMenuStrip groupedContextMenuStrip = new();
+
+        groupedContextMenuStrip.Populate();
+
+        groupedContextMenuStrip.Items.Count.Should().Be(0);
+    }
+
+    [Fact]
+    public void Populate_WhenInitializedWithGroup_ItemsCountIsNotZero()
+    {
+        GroupedContextMenuStrip groupedContextMenuStrip = new();
+        ToolStripButton toolStripButton = new();
```

Comment:
Ditto here and test below as well
```suggestion
        using GroupedContextMenuStrip groupedContextMenuStrip = new();
        using ToolStripButton toolStripButton = new();
```